doozerlib/distgit.py

```diff
@@ -196,7 +199,7 @@ def clone(self, distgits_root_dir, distgit_branch):
                     timeout = str(self.runtime.global_opts['rhpkg_clone_timeout'])
                     rhpkg_clone_depth = int(self.runtime.global_opts.get('rhpkg_clone_depth', '0'))
 
-                    if self.metadata.namespace == 'containers' and self.runtime.cache_dir:
+                    if self.metadata.namespace == 'containers':
```

Comment:
Currently, if cache_dir is used, we will clone the distgit repo with `git` instead of `rhpkg`. On buildvm, cache dir is used. Removing condition `and self.runtime.cache_dir` has no impact on buildvm but makes local runs more consistent with buildvm.